scripts/uberenv/spack_configs/linux_ubuntu_20/packages.yaml

```diff
@@ -85,6 +85,12 @@ packages:
     externals:
     - spec: doxygen
       prefix: /usr
+  llvm:
```

Comment:
I need to test this, but I'm pretty sure this will work.

Reply:
Yes.. you beat me to the comment of please make sure that these changes don't break your development machines I don't have access to.

Reply:
Might need an extra `apt-get` or it might build the `llvm` spack package and I'm not sure how large that is.

Reply:
works on my machine.